src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.MyAccount/ReturningCustomer/ReturningCustomerController.ts

```diff
@@ -98,9 +102,9 @@ module Orckestra.Composer {
          * Register Format validation to hide those server message on client interaction
          * Reset potentially unsafe fields
          */
-        private renderFailedForm(status: string) {
+        private renderFailedForm(status: string, accountLockedDownUntil: string = "") {
```

Comment:
just have here errorBag parameter
private renderFailedForm(status: string, errorBag : any = {}) {
this.render('ReturningCustomerFormsServerValidations', { Status: status, Bag: errorBag  });
...

}

in handlebar try to get needed value